pkg/policy/directory/watcher.go

```diff
@@ -124,7 +122,7 @@ func (p *policyWatcher) deleteFromPolicyEngine(cnpFilePath string) error {
 	cnp := p.fileNameToCnpCache[fileName]
 	if cnp == nil {
 		p.log.WithField("file", fileName).Error("BUG: Policy deletion request for file which was never added")
```

Comment:
we're now logging this error twice (once here and once at the call-site). If we return the error here, we probably can get rid of this log statement